examples/kubernetes/deployment.yml

```diff
@@ -23,7 +23,7 @@ spec:
         - /bin/elasticsearch_exporter
         - --es.uri=http://elasticsearch:9200
         - --es.all
-        image: quay.io/prometheuscommunity/elasticsearch-exporter:1.1.0
```

Comment:
This should be `latest` so that we do not need to update this example on every release.

Reply:
done